flycheck.el

```diff
@@ -3685,7 +3690,16 @@ Intended for use with `next-error-function'."
   (let ((pos (flycheck-next-error-pos n reset)))
     (if pos
         (goto-char pos)
-      (user-error "No more Flycheck errors"))))
+      (let ((result (cl-remove-if
```

Comment:
I don't know those newfangled things. Use whatever trivial list functions you like.